recommendation/static/version.json

```diff
@@ -0,0 +1,5 @@
+{
+    "commit": null,
+    "version": null,
+    "source": null
+}
```

Comment:
This is sort of random; CI will overwrite it, but it's helpful for local dev and testing, and seemed a little more meaningful than an empty object for the purposes of testing the view.